flamegraph/README.md

```diff
@@ -11,7 +11,9 @@ $ cd regex
 
 $ cargo rustc -- -Z self-profile
 
-$ flamegraph pid-{pid}
+$ cargo install --git https://github.com/rust-lang/measureme flamegraph
```

Comment:
I personally would move the installation command to the top of this snippet (with a comment similar to the one above), even before `git clone ...`